documentation/development.md

```diff
@@ -0,0 +1,60 @@
+# Git Town Development
+
+## Requirements
+
+* Ruby 2.1.5
+  (install [directly](https://www.ruby-lang.org/en/documentation/installation),
+  or via a ruby manager like [rvm](https://rvm.io/)
+  or [rbenv](https://github.com/sstephenson/rbenv))
+* [ShellCheck](https://github.com/koalaman/shellcheck)
+
+
+## Setup
+
+* fork and clone the repository to your machine
+* `bundle` to install ruby gems
+
+
+## Running Tests
+
+* Tests are written in [Cucumber](http://cukes.info/) and [RSpec](http://rspec.info/).
+* All features have need to have comprehensive test coverage
+* We are using linters for both the source and test files
+
+```bash
+# rake tasks
+rake          # Run linters and specs
+rake format   # Run formatters (fixes some lint errors)
+rake lint     # Run linters
+rake spec     # Run specs
+
+# run single test
```

Comment:
You use "spec" and "test" interchangeably here. I am guilty of this as well. Can we go with one only? I vote for "spec", but don't feel strongly about it.